packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+};
+
+const TSRules = {
+  "@typescript-eslint/adjacent-overload-signatures": "error",
+  "@typescript-eslint/array-type": "error",
+  "@typescript-eslint/ban-types": "error",
+  "@typescript-eslint/class-name-casing": "error",
+  "@typescript-eslint/consistent-type-assertions": "error",
+  "@typescript-eslint/consistent-type-definitions": "off",
+  "@typescript-eslint/explicit-member-accessibility": [
+    "off",
+    {
+      accessibility: "explicit"
+    }
+  ],
+  "@typescript-eslint/indent": "off",
+  "@typescript-eslint/interface-name-prefix": "off",
+  "@typescript-eslint/member-delimiter-style": [
+    "off",
+    "error",
+    {
+      multiline: {
+        delimiter: "none",
+        requireLast: true
+      },
+      singleline: {
+        delimiter: "semi",
+        requireLast: false
+      }
+    }
+  ],
+  "@typescript-eslint/member-ordering": "error",
+  "@typescript-eslint/no-empty-function": "error",
+  "@typescript-eslint/no-empty-interface": [
+    "error",
+    {
+      allowSingleExtends: true
+    }
+  ],
+  "@typescript-eslint/no-explicit-any": "off",
+  "@typescript-eslint/no-misused-new": "error",
+  "@typescript-eslint/no-namespace": "error",
+  "@typescript-eslint/no-parameter-properties": "off",
+  "@typescript-eslint/no-use-before-declare": "off",
+  "@typescript-eslint/no-var-requires": "error",
+  "@typescript-eslint/prefer-for-of": "off",
+  "@typescript-eslint/prefer-function-type": "error",
+  "@typescript-eslint/prefer-namespace-keyword": "error",
+  "@typescript-eslint/quotes": "off",
+  "@typescript-eslint/semi": ["off", null],
+  "@typescript-eslint/space-within-parens": ["off", "never"],
+  "@typescript-eslint/triple-slash-reference": "error",
+  "@typescript-eslint/type-annotation-spacing": "off",
+  "@typescript-eslint/unified-signatures": "error",
+  "@typescript-eslint/explicit-function-return-type": "off",
+  "@typescript-eslint/no-unused-vars": "off",
+  "@typescript-eslint/ban-ts-ignore": "off",
+  "@typescript-eslint/no-use-before-define": "off",
+  "@typescript-eslint/camelcase": "off"
+};
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
+    ecmaVersion: 2018,
+    sourceType: "module"
+  },
+  plugins: ["@typescript-eslint", "@typescript-eslint/tslint", "prettier"],
+  extends: [
+    "plugin:@typescript-eslint/recommended",
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript",
+    "plugin:react/recommended",
+    "prettier/@typescript-eslint",
+    "prettier"
+  ],
+  rules: Object.assign(rules, TSRules),
+  settings: {
+    "import/ignore": ["node_modules"],
+    react: {
+      version: "detect"
+    }
+  },
+  globals: {
+    // Maybe we should be importing these?
+    ethereum: "readonly",
+    web3: "readonly",
+    process: "readonly"
+  },
```

Comment:
Why do `globals` need to be defined twice? We also define them in `globals.d.ts` for example.

Reply:
We define them as vars. This allows Typescript to understand them. ESLint though doesn't understand that they are globals.